doc/reference/dune-workspace/lock_dir.rst

```diff
@@ -0,0 +1,72 @@
+lock_dir
+========
+
+.. warning::
+
+   Dune Package Management is not final yet and the configuration options
+   are subject to change.
+
+This stanza configures settings for lock directories what are relevant to
```

Comment:
```suggestion
This stanza configures settings for lock directories who are relevant to
```
Otherwise, I'm not sure to get the meaning of the sentence :sweat: 

Shouldn't we mention that this is related to package management (apart from the warning)?